numpy/testing/_private/utils.py

```diff
@@ -699,7 +699,8 @@ def assert_approx_equal(actual,desired,significant=7,err_msg='',verbose=True):
 
 
 def assert_array_compare(comparison, x, y, err_msg='', verbose=True, header='',
-                         precision=6, equal_nan=True, equal_inf=True):
+                         precision=6, equal_nan=True, equal_inf=True,
+                         strict=False):
```

Comment:
```suggestion
                         *, strict=False):
```
New optional arguments are generally made keyword-only these days,